controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -303,28 +303,28 @@ def test_get_volume_mappings(self):
     def test_map_volume_host_not_found(self):
         self.client_mock.map_volume_to_host.side_effect = NotFound("404")
         with self.assertRaises(array_errors.HostNotFoundError):
-            self.array.map_volume("fake_name", "fake_host")
+            self.array.map_volume("fake_name", "fake_host", "fake_connectivity_type")
 
     def test_map_volume_volume_not_found(self):
         self.client_mock.map_volume_to_host.side_effect = ClientException("500", "[BE586015]")
         with self.assertRaises(array_errors.ObjectNotFoundError):
-            self.array.map_volume("fake_name", "fake_host")
+            self.array.map_volume("fake_name", "fake_host", "fake_connectivity_type")
 
     def test_map_volume_no_available_lun(self):
         self.client_mock.map_volume_to_host.side_effect = InternalServerError("500", "[BE74121B]")
         with self.assertRaises(array_errors.NoAvailableLunError):
-            self.array.map_volume("fake_name", "fake_host")
+            self.array.map_volume("fake_name", "fake_host", "fake_connectivity_type")
 
     def test_map_volume_fail_with_ClientException(self):
         self.client_mock.map_volume_to_host.side_effect = ClientException("500")
         with self.assertRaises(array_errors.MappingError):
-            self.array.map_volume("fake_name", "fake_host")
+            self.array.map_volume("fake_name", "fake_host", "fake_connectivity_type")
 
     def test_map_volume(self):
         scsi_id = "6005076306FFD3010000000000000001"
         host_name = "test_name"
         self.client_mock.map_volume_to_host.return_value = Munch({"lunid": "01"})
-        lun = self.array.map_volume(scsi_id, host_name)
+        lun = self.array.map_volume(scsi_id, host_name, "fake_connectivity_type")
```

Comment:
it seems in the function the argument values should all be defined in variables before the call

Reply:
done